### PR TITLE
VS Tizen extension user docs update for .Net and Hybrid App.

### PR DESCRIPTION
**Pull request by om90prakash**
Signed-off-by: Om Prakash/Things Platform /SRI-Bangalore/Engineer/Samsung Electronics <om90.prakash@samsung.com>

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Review by sangwook96 — APPROVED** [review]
Thank you very much  !!

**Review by safir-srbd — COMMENTED** [review]
@om90prakash Hello. Hope you are well. I have reviewed this PR, and have suggested some changes. Please incorporate them into the PR.

File: docs/application/vstools/Tizen/hybrid.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+4. In the **Tizen Profile Select** window, select the required template for your project, then click **OK**.
+   
+   ![Version selection](media/native_platform_version.PNG)
+
+5. The visual studio window with newly created project appears on the **Solution Explorer**.
+
+   ![Visual Studio screen](media/hybrid_vs_screen.PNG)
+
+
+### Set Startup Project
```
Comment (safir-srbd) [inline docs/application/vstools/Tizen/hybrid.md:37]: Headings should follow sentence style case.
Ref: https://github.com/Samsung/tizen-docs/blob/master/reviewguide/review_points_guide.md#headings
```suggestion
### Set startup project
```

File: docs/application/vstools/Tizen/hybrid.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+1.  From **Solution Explorer** Right click on the desired Project and select **Set as Startup Project**.
+
+
+### Add Project Dependency
```
Comment (safir-srbd) [inline docs/application/vstools/Tizen/hybrid.md:42]: Headings should follow sentence style case.
Ref: https://github.com/Samsung/tizen-docs/blob/master/reviewguide/review_points_guide.md#headings
```suggestion
### Add project dependency
```

**Review by safir-srbd — COMMENTED** [review]
@om90prakash I have added a few more changes.

File: docs/application/vstools/Tizen/hybrid.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+   ![Version selection](media/native_platform_version.PNG)
+
+5. The Visual Studio window with newly created project appears on the **Solution Explorer**.
+
+   ![Visual Studio screen](media/hybrid_vs_screen.PNG)
+
+
+### Set startup project
+
+1.  From **Solution Explorer** right click on the desired project and select **Set as Startup Project**.
```
Comment (safir-srbd) [inline docs/application/vstools/Tizen/hybrid.md:39]: Added a missing comma to improve readability.
```suggestion
1.  From **Solution Explorer**, right click on the desired project and select **Set as Startup Project**.
```

File: docs/application/vstools/Tizen/hybrid.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+
+### Add project dependency
+
+1.  From Solution Explorer right click on **Startup Project** and select **Add Tizen Project Dependency**.
```
Comment (safir-srbd) [inline docs/application/vstools/Tizen/hybrid.md:44]: Added a missing comma to improve readability.
```suggestion
1.  From Solution Explorer, right click on **Startup Project** and select **Add Tizen Project Dependency**.
```

**Review by safir-srbd — COMMENTED** [review]
@om90prakash Thank you for incorporating the suggested changes. It appears you have missed out on two small suggestions, so I have made those suggestions again. Please incorporate them into this PR.

**Review by safir-srbd — APPROVED** [review]
@om90prakash Thank you for incorporating all suggested changes.

@mijin85cho @asm-foysal I am approving this PR for merging.

**Pull request merged**